src/docfx/cli/Program.cs

```diff
@@ -9,23 +10,43 @@ internal class Program
     {
         internal static async Task Main(string[] args)
         {
-            var (command, docset, options) = ParseCommandLineOptions(args);
+            var (command, source, options) = ParseCommandLineOptions(args);
             var log = new ConsoleLog();
 
             switch (command)
             {
                 case "restore":
-                    await Restore.Run(docset, options, log);
+                    await Restore.Run(source, options, log);
                     break;
                 case "build":
-                    await Build.Run(docset, options, log);
+                    await Build.Run(source, options, log);
                     break;
             }
         }
 
-        private static (string command, string docset, CommandLineOptions options) ParseCommandLineOptions(string[] args)
+        private static (string command, string source, CommandLineOptions options) ParseCommandLineOptions(string[] args)
         {
-            return (args[0], args[1], default);
+            var command = string.Empty;
+            var source = ".";
+            var options = new CommandLineOptions();
+
+            ArgumentSyntax.Parse(args, syntax =>
+            {
+                // Restore command
+                syntax.DefineCommand("restore", ref command, "restores dependencies before build");
```

Comment:
>DefineCommand [](start = 23, length = 13)

would you please help to add a example about how to use this command to the comments?